### PR TITLE
rename hash -> HASHING_ALGORITHM to be more descriptive and use keccac to make it verfiable

### PR DESCRIPTION
**Pull request by frankiebee**
This pr is so that the cli can verify the signature against the verifying key using keccac.
This is also renames hash to be more understandable `HASHING_ALGORITHM`

### CODE REVIEW

File: src/signing/adapters/base.ts [hunk 1]
```diff
@@ -59,4 +59,4 @@ export async function preSign(
 
 export const type = 'noop'
 // THIS IS THE RETURNED SIG HASH -- PLS look into ED25519 ASK JESSE IF SUPPORTED
-export const hash = 'sha3'
+export const HASHING_ALGORITHM = 'sha3'
```
Comment (mixmix) [inline src/signing/adapters/base.ts:62]: :heart: :boom:

File: src/signing/adapters/types.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export interface PRESIGN_RESULT {
  */
 export interface Adapter {
   type: string
-  hash: string
+  HASHING_ALGORITHM: string
```
Comment (mixmix) [inline src/signing/adapters/types.ts:28]: :fire: - I am pretty sure this is a mistake?
Out lack of testing in this module is _dangerous_

```suggestion
  hash: string
```

**Review by mixmix — CHANGES_REQUESTED** [review]
Looks good but for one woops?

File: src/signing/index.ts [hunk 3]
```diff
@@ -147,7 +147,7 @@ export default class SignatureRequestManager {
     const { sigRequestHash } = results[0]
     const signature = await this.sign({
       sigRequestHash,
-      hash: adaptersToRun[0].hash,
+      hash: adaptersToRun[0].HASHING_ALGORITHM,
```
Comment (rh0delta) [inline src/signing/index.ts:150]: why do we want to leave the argument as `hash` and only change the keyword in the adapters?

File: src/signing/adapters/types.ts (hunk 2 above)
Comment (frankiebee) [reply src/signing/adapters/types.ts:28]: tldr; this is a documentation issue not a code one

**Review by rh0delta — APPROVED** [review]

File: src/signing/index.ts (hunk 3 above)
Comment (frankiebee) [reply src/signing/index.ts:150]: one thing at a time. signing file is protected space

**Pull request merged**